pkg/v1/providers/tests/unit/avi_test.go

```diff
@@ -190,6 +190,7 @@ var _ = Describe("AKO-operator Ytt Templating", func() {
 			"VSPHERE_FOLDER":             "ds0",
 			"VSPHERE_SSH_AUTHORIZED_KEY": "ssh-rsa AAAA...+M7Q== vmware-tanzu.local",
 			"VSPHERE_INSECURE":           "true",
+			"CLUSTER_CIDR":               "192.168.1.0/16",
```

Comment:
Trying to understand why the value for AVI tests should be modified? any particular reasons here?

Reply:
The 1.22.5 vsphere-cpi brings schema validation. The [cpi_addon_data.lib.yaml](https://github.com/vmware-tanzu/tanzu-framework/blob/main/pkg/v1/providers/ytt/02_addons/cpi/cpi_addon_data.lib.yaml#L27) file renders config that requires this value. Because the avi tests are rendering all of the templates, they were affected by the bump.